client/js/render.js

```diff
@@ -0,0 +1,192 @@
+"use strict";
+const $ = require("jquery");
+const templates = require("../views");
+const options = require("./options");
+const utils = require("./utils");
+const sorting = require("./sorting");
+
+const chat = $("#chat");
+const sidebar = $("#sidebar, #footer");
```

Comment:
This is probably extracted from `lounge.js`, but do we actually want to keep both selectors in there?